src/pages/Check.jsx

```diff
@@ -6,15 +6,23 @@ import { Button, Input, Modal, Text } from '../components'
 import ticket from '../assets/ticket.png'
 import warning from '../assets/warning.png'
 import arrow from '../assets/arrow-left.svg'
-
+import { useGetPackageInformation } from '../hooks'
 const Check = () => {
   const [packageCode, setPackageCode] = useState('')
   const [isModalOpen, changeIsModalOpen] = useState(true)
   const history = useHistory()
+  const { packageInformation } = useGetPackageInformation({ packageId: packageCode })
 
-  const toDelivery = e => {
+  const toDelivery = async e => {
     e.preventDefault()
-    history.push('/delivery/test')
+    const { packages } = await packageInformation
+    if ((await packages.length) > 0) {
+      history.push(`/delivery/${packageCode}`)
+    }
+
+    if ((await packages.length) === 0) {
+      alert('el packcode no existe')
+    }
```

Comment:
What about
```suggestion
    if ((await packages.length) > 0) {
      history.push(`/delivery/${packageCode}`)
    }

    alert('el packcode no existe')
```

The idea is that the IF only can have 2 results, and your second conditional is precisely the other result, so is not really needed to create another IF block